subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -165,6 +165,7 @@ config BT_TBS_CLIENT_MAX_CALLS
 config BT_TBS_CLIENT_MAX_TBS_INSTANCES
 	int "Maximum number of TBS instances to setup"
 	default 1
+	range 0 3 if BT_TBS_CLIENT_GTBS
 	range 1 3
```

Comment:
We can make the range unconditionally 0-3 (or higher) if we decided to always require `BT_TBS_CLIENT_GTBS=y`. 

In other words, apply the same requirement to support GTBS for the client, as the server has. 

That would essentially imply that `BT_TBS_CLIENT_GTBS` is always true, and can be removed.

Reply:
According to CCP specification, it's up to implementation to use GTBS or TBS.
`Call Control Clients can access TBS, GTBS, or both on a Call Control Server.`
So I guess we cannot imply using GTBS by default

Reply:
But is there a real use case where we want to to have a client implementation that only supports discovering TBS, and not GTBS?

Reply:
I guess there might be. e.g. Client that can control Skype only

Reply:
Good point :) 

We can leave it as it then